recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,49 @@
+c_compiler:                    # [win]
+  - vs2008                     # [win]
+  - vs2015                     # [win]
+  - vs2015                     # [win]
+cxx_compiler:                  # [win]
+  - vs2008                     # [win]
+  - vs2015                     # [win]
+  - vs2015                     # [win]
+CONDA_BUILD_SYSROOT:           # [osx]
+  - /opt/MacOSX10.9.sdk        # [osx]
+VERBOSE_AT:
+  - V=1
+VERBOSE_CM:
+  - VERBOSE=1
```

Comment:
Note to self. Read the docs on what this entails or hope for an expert to comment here after me :stuck_out_tongue_closed_eyes:

Reply:
Ray has added these to switch verbosity for autotools (AT) and cmake (CM).

Reply:
conda-build 3 just sets an env var (if one isn't already set) for these and I use them directly in the recipes:

For ones that use autotools:

`make -j${CPU_COUNT} ${VERBOSE_AT}`

For ones that use cmake:

`make -j${CPU_COUNT} ${VERBOSE_CM}`

The reason for this is to allow you guys to remove it if it causes problems problems on CI with large log sizes, but I am very much in the 'log everything by default' camp, because otherwise build logs are pretty useless.

Reply:
This is awesome @mingwandroid. We can really simplify a lot of our workflow with this approach while keeping the recipes "clean" and re-usable.